README.md

```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
 
 ### Overview
 This repo contains tools to help integrate Roslyn analyzers with SonarQube so that issues detected by the Roslyn analyzers are reported and managed in SonarQube.
-Specifically, the tools will generate a Java SonarQube plugin that registers the rules with SonarQube. The generated plugin works with the [C# plugin](http://docs.sonarqube.org/x/bAAW) (v4.5 or higher) and the [SonarQube Scanner for MSBuild](http://docs.sonarqube.org/x/Lx9q) (v2.0 or higher) to handle executing the analyzer and uploading any issues.
+Specifically, the tools will generate a Java SonarQube plugin that registers the rules with SonarQube. The generated plugin works with the [C# plugin](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/languages/csharp/) (v4.5 or higher) and the [SonarQube Scanner for MSBuild](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/) (v2.0 or higher) to handle executing the analyzer and uploading any issues.
```

Comment:
I would probably remove the `(v2.0 or higher)` commitment for the backward compatibility. 
I don't think users really care, and we want everyone to be on the latest anyway